core/tests/widgets/test_webview.py

```diff
@@ -183,6 +183,27 @@ def test_set_content(widget):
     )
 
 
+def test_init_widget_with_content():
```

Comment:
The convention in the tests is that constructor tests are always the first tests in the file. These two cases should be up around L55, after `test_create_with_values` - and should be named to match.